skimage/filters/thresholding.py

```diff
@@ -578,57 +578,43 @@ def threshold_minimum(image, nbins=256, bias='min', max_iter=10000):
     >>> binary = image > thresh
     """
 
-    def find_local_maxima(hist):
+    def find_local_maxima_idx(hist):
```

Comment:
Actually, I think that they *can't* happen, *unless* the image has all equal values. Otherwise the worst case, I think, is that you need to do a few more smoothing iterations. But eventually, you'll end up with a maximum at the centre of the plateau. Thoughts?